### DISCUSSION TITLE
[solved] quarkus-scheduler-2.8.0.Final.pom not available in repo1.maven.org

### DISCUSSION THREAD

**Discussion by mickroll** [quarkusio/quarkus#24689] [Community]
It seems that the upload of `2.8.0.Final` is broken in http://repo1.maven.org.

https://repo1.maven.org/maven2/io/quarkus/quarkus-scheduler/2.8.0.Final/ shows that its pom is available, but if i try to download `quarkus-scheduler-2.8.0.Final.pom` i get a 
> 404 Not Found

Is there a way to fix that? Maybe reupload or contacting the repository staff?

Details:
![grafik](https://user-images.githubusercontent.com/9883575/161218047-a45dad1e-67de-4f1e-83f0-10ee59e5dd3f.png)

```
<html>
<head><title>404 Not Found</title></head>
<body bgcolor="white">
<center><h1>404 Not Found</h1></center>
<hr><center>nginx</center>
</body>
</html>
```
... unfortunately not very helpful.

**Comment by mickroll** [comment]
Btw. there are no release notes on 2.8.0.Final, only for 2.8.0.CR.

**Comment by gsmet** [comment]
See https://groups.google.com/g/quarkus-dev/c/lQoIO1LduyU

>  there are no release notes on 2.8.0.Final, only for 2.8.0.CR.

That's expected, 2.8.0.Final has not been released yet.

**Reply by mickroll** [reply]
> See https://groups.google.com/g/quarkus-dev/c/lQoIO1LduyU
> 
> > there are no release notes on 2.8.0.Final, only for 2.8.0.CR.
> 
> That's expected, 2.8.0.Final has not been released yet.

So quarkus-scheduler was released early. OK, i didn't know some artifacts are released separately.

**Reply by gsmet** [reply]
No. It wasn't. The release is in limbo right now with the artifacts partially synced to Central.

And in any case, the release is announced after the Platform release which is one week after the Core release so there's always a one week delay between the artifacts hitting Central and the announcement.

**Reply by mickroll** [reply]
Partial sync makes sense. We'll wait for it to complete

Thanks!

**Reply by gsmet** [reply]
I'll post here when the issue is resolved. For now I have no idea when it will be :/

**Reply by famod** [reply]
Looking better now!

**Reply by kahowell** [reply]
FYI, there are still some inconsistencies. For example: https://repo.maven.apache.org/maven2/io/quarkus/platform/quarkus-bom/maven-metadata.xml does not show 2.8.0.Final yet.

**Reply by gsmet** [reply]
@kahowell no, that's normal, we haven't released the Platform yet. Quarkus 2.8.0.Final is not officially released. We usually have a one week delay between the core artifacts and the full platform release to let the time to the platform members to push their updates but this time, we had to wait for more given the sync issue. I plan to release 2.8.0.Final officially on Tuesday.
